raft-tech/TANF-app
# feat/1957 include current fiscal year

## Summary of Changes
Pull request closes #1957

## How to Test

```
cd tdrs-frontend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d
cd tdrs-backend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d 
```

1. Open http://localhost:3000/ and sign in.
2. Go to `Data Files` tab.
3. Open `Select Fiscal Year` drop down and see that 2022 is now present

![Screen Shot 2022-07-20 at 12 50 46 PM](https://user-images.githubusercontent.com/7119690/180069462-64f549bc-afca-40b3-8cfe-8cf16d443ea5.png)

The following screen shot shows that when the month is after September the next fiscal year will be added to the dropdown.
![Screen Shot 2022-07-20 at 12 17 31 PM](https://user-images.githubusercontent.com/7119690/180065364-051ee988-342a-4eab-ba37-e9bea5e6b116.png)



## Deliverables
_More details on how deliverables herein are assessed included [here](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverables)._

### [Deliverable 1: Accepted Features](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-1-Accepted-Features)

**Acceptance Criteria:** 
- [x] Users can submit data files for FY2022

**Tasks:**
- [x] update source of data_file `year` metadata for dropdown menu to include 2022

### [Deliverable 3: Properly Styled Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-3-Properly-Styled-Code)


### [Deliverable 4: Accessible](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)



### [Deliverable 5: Deployed](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-5-Deployed)


### [Deliverable 6: Documented](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-6-Code-documentation)


### [Deliverable 7: Secure](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-7-Secure)


### [Deliverable 8: User Research](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-8-User-Research)

Research product(s) clearly articulate(s):
